### PR TITLE
[ServiceBus] Fix typos

### PR DESCRIPTION
**Pull request by MiYanni**
Had it search for typos in the solution. Then, I went through them and fixed them.

And yes, it is `dead-letter`: https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-dead-letter-queues

### CODE REVIEW

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/Rules/CorrelationRuleFilter.cs [hunk 1]
```diff
@@ -25,7 +25,7 @@ namespace Azure.Messaging.ServiceBus.Administration
     /// </para>
     /// <para>
     /// The CorrelationRuleFilter provides an efficient shortcut for declarations of filters that deal only with correlation equality.
-    /// In this case the cost of the lexigraphical analysis of the expression can be avoided.
+    /// In this case the cost of the lexicographical analysis of the expression can be avoided.
```
Comment (jsquire) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/Rules/CorrelationRuleFilter.cs:28]: I'll admit.... I had to look this up and compare the terms since both are valid spellings.   Nice catch!

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpConnectionScope.cs [hunk 2]
```diff
@@ -823,7 +823,7 @@ public override void Dispose()
         /// <param name="entityPath"></param>
         ///
         /// <param name="connection">The AMQP connection to which the link being refreshed is bound to.</param>
-        /// <param name="amqpLink">The AMQO link to refresh authorization for.</param>
+        /// <param name="amqpLink">The AMQP link to refresh authorization for.</param>
```
Comment (jsquire) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpConnectionScope.cs:826]: $1 says that I was responsible for this one.....   😛

**Review by jsquire — APPROVED** [review]

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/Rules/CorrelationRuleFilter.cs (hunk 1 above)
Comment (MiYanni) [reply sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/Rules/CorrelationRuleFilter.cs:28]: I also had to look it up. I've heard 'lexigraphical' say verbally, but seen 'lexicographical' written before. Most dictionaries don't have 'lexigraphical'.

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/ServiceBusAdministrationClient.cs [hunk 3]
```diff
@@ -49,7 +49,7 @@ public class ServiceBusAdministrationClient
         private const string RulesPath = "{0}/Subscriptions/{1}/rules";
 
         /// <summary>
-        /// Parameterless constructor to allow mocking.
+        /// Parameter-less constructor to allow mocking.
```
Comment (JoshLove-msft) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/ServiceBusAdministrationClient.cs:52]: Why this change?

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs [hunk 4]
```diff
@@ -441,7 +441,7 @@ private static void CloseLink(RequestResponseAmqpLink link)
                 if (exception is OperationCanceledException &&
                     receiveLink != null && receiveLink.State != AmqpObjectState.Opened)
                 {
-                    // The link state is lost, We need to return a non-retriable error.
+                    // The link state is lost, We need to return a non-retrievable error.
```
Comment (JoshLove-msft) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs:444]: Why? non-retriable is correct.

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/ServiceBusAdministrationClient.cs (hunk 3 above)
Comment (MiYanni) [reply sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/ServiceBusAdministrationClient.cs:52]: I'll revert those with this change. It doesn't exist in the dictionary used for validation, but it clearly understood as a word at this point in computer science.
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/using-constructors

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs (hunk 4 above)
Comment (MiYanni) [reply sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs:444]: I literally couldn't parse the word, so I misunderstood what the intent was. My bad. I've never seen it written like that. Seems that it is more popular in our culture now.
https://english.stackexchange.com/a/305433

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpConnectionScope.cs (hunk 2 above)
Comment (MiYanni) [reply sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpConnectionScope.cs:826]: 😝

File: sdk/core/Azure.Core/src/RetryOptions.cs [hunk 5]
```diff
@@ -22,7 +22,7 @@ internal RetryOptions()
 
         /// <summary>
         /// The delay between retry attempts for a fixed approach or the delay
-        /// on which to base calculations for a backoff-based approach.
+        /// on which to base calculations for a back-off-based approach.
```
Comment (JoshLove-msft) [inline sdk/core/Azure.Core/src/RetryOptions.cs:25]: double hyphen? 👀

**Comment by JoshLove-msft** [comment]
@pakrym - this has some doc fixes for Core as well.

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/ServiceBusAdministrationClient.cs (hunk 3 above)
Comment (jsquire) [reply sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/ServiceBusAdministrationClient.cs:52]: 😄    I wasn't going to pick on it, and it is grammatically correct, but I really didn't like the hyphen.

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs (hunk 4 above)
Comment (jsquire) [reply sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs:444]: Whooops.   I totally missed that.   Nice catch, Josh.

File: sdk/core/Azure.Core/src/RetryOptions.cs (hunk 5 above)
Comment (jsquire) [reply sdk/core/Azure.Core/src/RetryOptions.cs:25]: image

Comment (JoshLove-msft) [reply sdk/core/Azure.Core/src/RetryOptions.cs:25]: I was thinking of a double guitar 😆

File: sdk/core/Azure.Core.TestFramework/src/RecordEntry.cs [hunk 6]
```diff
@@ -17,7 +17,7 @@ public class RecordEntry
         private static readonly JsonWriterOptions RequestWriterOptions = new JsonWriterOptions();
         // Responses are usually formatted using Newtonsoft.Json that has more relaxed encoding rules
         // To enable us to store more responses as JSON instead of string in Recording files use
-        // relaxed settings for roundrip
```
Comment (JoshLove-msft) [inline sdk/core/Azure.Core.TestFramework/src/RecordEntry.cs:20]: 😆

File: sdk/core/Azure.Core/src/Pipeline/Internal/HttpEnvironmentProxy.cs [hunk 7]
```diff
@@ -210,7 +210,7 @@ private bool IsMatchInBypassList(Uri input)
                 {
                     if (s[0] == '.')
                     {
-                        // This should match either domain it self or any subdomain or host
+                        // This should match either domain it self or any sub-domain or host
```
Comment (JoshLove-msft) [inline sdk/core/Azure.Core/src/Pipeline/Internal/HttpEnvironmentProxy.cs:213]: I think subdomain is correct:
https://en.wikipedia.org/wiki/Subdomain

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpMessageConstants.cs [hunk 8]
```diff
@@ -24,14 +24,14 @@ internal class AmqpMessageConstants
...
-        ///  Property key representing deadletter reason, when a message is received from a deadletter subqueue of an entity.
+        ///  Property key representing dead-letter reason, when a message is received from a dead-letter sub-queue of an entity.
         ///  This key and the associated values are stored in the <see cref="ServiceBusReceivedMessage.ApplicationProperties"/> dictionary
         ///  for dead lettered messages.
         /// </summary>
         internal const string DeadLetterReasonHeader = "DeadLetterReason";
 
         /// <summary>
-        ///  Property key representing detailed error description, when a message is received from a deadletter subqueue of an entity.
+        ///  Property key representing detailed error description, when a message is received from a dead-letter sub-queue of an entity.
```
Comment (JoshLove-msft) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpMessageConstants.cs:34]: subqueue matches the service documentation - https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-dead-letter-queues

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/ServiceBusAdministrationClient.cs (hunk 3 above)
Comment (MiYanni) [reply sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/ServiceBusAdministrationClient.cs:52]:
```suggestion
        /// Parameterless constructor to allow mocking.
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Core/CancellationTokenExtensions.cs [hunk 9]
```diff
@@ -18,7 +18,7 @@ internal static class CancellationTokenExtensions
         ///   of the <paramref name="instance" />.
         /// </summary>
         ///
-        /// <typeparam name="T">The type of exception to throw; the type must have a parameterless constructor.</typeparam>
+        /// <typeparam name="T">The type of exception to throw; the type must have a parameter-less constructor.</typeparam>
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Core/CancellationTokenExtensions.cs:21]:
```suggestion
        /// <typeparam name="T">The type of exception to throw; the type must have a parameterless constructor.</typeparam>
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Diagnostics/ServiceBusEventSource.cs [hunk 10]
```diff
@@ -37,13 +37,13 @@ internal class ServiceBusEventSource : EventSource
...
         ///   created outside the scope of the <see cref="Log" /> instance, as well as setting up the
-        ///   integration with AzureEventSourceListenter.
+        ///   integration with AzureEventSourceListener.
         /// </summary>
         private ServiceBusEventSource(string eventSourceName) : base(eventSourceName, EventSourceSettings.Default, AzureEventSourceListener.TraitName, AzureEventSourceListener.TraitValue)
         {
         }
 
-        // parameterless constructor for mocking
+        // parameter-less constructor for mocking
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Diagnostics/ServiceBusEventSource.cs:46]:
```suggestion
        // Parameterless constructor for mocking
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ServiceBusConnection.cs [hunk 11]
```diff
@@ -61,7 +61,7 @@ internal class ServiceBusConnection : IAsyncDisposable
         private readonly TransportClient _innerClient;
 
         /// <summary>
-        /// Parameterless constructor to allow mocking.
+        /// Parameter-less constructor to allow mocking.
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ServiceBusConnection.cs:64]:
```suggestion
        /// Parameterless constructor to allow mocking.
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs (hunk 4 above)
Comment (MiYanni) [reply sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs:444]:
```suggestion
                    // The link state is lost, We need to return a non-retriable error.
```

File: sdk/core/Azure.Core/src/RetryMode.cs [hunk 12]
```diff
@@ -15,7 +15,7 @@ public enum RetryMode
         Fixed,
 
         /// <summary>
-        /// Retry attempts will delay based on a backoff strategy, where each attempt will increase
+        /// Retry attempts will delay based on a back-off strategy, where each attempt will increase
```
Comment (MiYanni) [inline sdk/core/Azure.Core/src/RetryMode.cs:18]:
```suggestion
        /// Retry attempts will delay based on a backoff strategy, where each attempt will increase
```

File: sdk/core/Azure.Core/src/RetryOptions.cs (hunk 5 above)
Comment (MiYanni) [reply sdk/core/Azure.Core/src/RetryOptions.cs:25]: I looked it up. You're right. The `backoff` mentioned here is related to this: https://en.wikipedia.org/wiki/Exponential_backoff
```suggestion
        /// on which to base calculations for a backoff-based approach.
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ServiceBusRetryOptions.cs [hunk 13]
```diff
@@ -49,7 +49,7 @@ public int MaxRetries
 
         /// <summary>
         ///   The delay between retry attempts for a fixed approach or the delay
-        ///   on which to base calculations for a backoff-based approach.
+        ///   on which to base calculations for a back-off-based approach.
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ServiceBusRetryOptions.cs:52]:
```suggestion
        ///   on which to base calculations for a backoff-based approach.
```

File: sdk/core/Azure.Core/src/Pipeline/Internal/HttpEnvironmentProxy.cs (hunk 7 above)
Comment (MiYanni) [reply sdk/core/Azure.Core/src/Pipeline/Internal/HttpEnvironmentProxy.cs:213]:
```suggestion
                        // This should match either domain it self or any subdomain or host
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpMessageConstants.cs [hunk 14]
```diff
@@ -24,14 +24,14 @@ internal class AmqpMessageConstants
         internal const string UriName = AmqpConstants.Vendor + ":uri";
         internal const string DateTimeOffsetName = AmqpConstants.Vendor + ":datetime-offset";
         /// <summary>
-        ///  Property key representing deadletter reason, when a message is received from a deadletter subqueue of an entity.
+        ///  Property key representing dead-letter reason, when a message is received from a dead-letter sub-queue of an entity.
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpMessageConstants.cs:27]:
```suggestion
        ///  Property key representing dead-letter reason, when a message is received from a dead-letter subqueue of an entity.
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpMessageConstants.cs (hunk 8 above)
Comment (MiYanni) [reply sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpMessageConstants.cs:34]:
```suggestion
        ///  Property key representing detailed error description, when a message is received from a dead-letter subqueue of an entity.
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs [hunk 15]
```diff
@@ -576,17 +576,17 @@ private void ThrowLockLostException()
         }
 
         /// <summary>
-        /// Moves a message to the deadletter sub-queue.
+        /// Moves a message to the dead-letter sub-queue.
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs:579]:
```suggestion
        /// Moves a message to the dead-letter subqueue.
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs [hunk 16]
```diff
@@ -576,17 +576,17 @@ private void ThrowLockLostException()
...
         /// </summary>
         ///
-        /// <param name="lockToken">The lock token of the <see cref="ServiceBusReceivedMessage"/> to deadletter.</param>
-        /// <param name="deadLetterReason">The reason for deadlettering the message.</param>
-        /// <param name="deadLetterErrorDescription">The error description for deadlettering the message.</param>
+        /// <param name="lockToken">The lock token of the <see cref="ServiceBusReceivedMessage"/> to dead-letter.</param>
+        /// <param name="deadLetterReason">The reason for dead-lettering the message.</param>
+        /// <param name="deadLetterErrorDescription">The error description for dead-lettering the message.</param>
         /// <param name="propertiesToModify">The properties of the message to modify while moving to sub-queue.</param>
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs:585]:
```suggestion
        /// <param name="propertiesToModify">The properties of the message to modify while moving to subqueue.</param>
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs [hunk 17]
```diff
@@ -611,14 +611,14 @@ private void ThrowLockLostException()
                 cancellationToken).ConfigureAwait(false);
 
         /// <summary>
-        /// Moves a message to the deadletter sub-queue.
+        /// Moves a message to the dead-letter sub-queue.
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs:614]:
```suggestion
        /// Moves a message to the dead-letter subqueue.
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs [hunk 18]
```diff
@@ -611,14 +611,14 @@ private void ThrowLockLostException()
...
         /// </summary>
         ///
-        /// <param name="lockToken">The lock token of the corresponding message to deadletter.</param>
+        /// <param name="lockToken">The lock token of the corresponding message to dead-letter.</param>
         /// <param name="timeout"></param>
         /// <param name="propertiesToModify">The properties of the message to modify while moving to sub-queue.</param>
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs:619]:
```suggestion
        /// <param name="propertiesToModify">The properties of the message to modify while moving to subqueue.</param>
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Core/TransportReceiver.cs [hunk 19]
```diff
@@ -146,17 +146,17 @@ internal abstract class TransportReceiver
             CancellationToken cancellationToken = default);
 
         /// <summary>
-        /// Moves a message to the deadletter sub-queue.
+        /// Moves a message to the dead-letter sub-queue.
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Core/TransportReceiver.cs:149]:
```suggestion
        /// Moves a message to the dead-letter subqueue.
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Core/TransportReceiver.cs [hunk 20]
```diff
@@ -146,17 +146,17 @@ internal abstract class TransportReceiver
...
         /// </summary>
         ///
-        /// <param name="lockToken">The lock token of the <see cref="ServiceBusReceivedMessage"/> to deadletter.</param>
-        /// <param name="deadLetterReason">The reason for deadlettering the message.</param>
-        /// <param name="deadLetterErrorDescription">The error description for deadlettering the message.</param>
+        /// <param name="lockToken">The lock token of the <see cref="ServiceBusReceivedMessage"/> to dead-letter.</param>
+        /// <param name="deadLetterReason">The reason for dead-lettering the message.</param>
+        /// <param name="deadLetterErrorDescription">The error description for dead-lettering the message.</param>
         /// <param name="propertiesToModify">The properties of the message to modify while moving to sub-queue.</param>
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Core/TransportReceiver.cs:155]:
```suggestion
        /// <param name="propertiesToModify">The properties of the message to modify while moving to subqueue.</param>
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/EntityNameFormatter.cs [hunk 21]
```diff
@@ -31,11 +31,11 @@ public static string FormatDeadLetterPath(string entityPath)
         }
 
         /// <summary>
-        /// Formats the subqueue path for either a queue, or a subscription.
+        /// Formats the sub-queue path for either a queue, or a subscription.
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/EntityNameFormatter.cs:34]:
```suggestion
        /// Formats the subqueue path for either a queue, or a subscription.
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/EntityNameFormatter.cs [hunk 22]
```diff
@@ -31,11 +31,11 @@ public static string FormatDeadLetterPath(string entityPath)
...
         /// </summary>
         /// <param name="entityPath">The name of the queue, or path of the subscription.</param>
         /// <param name="subQueueName"></param>
-        /// <returns>The path as a string of the subqueue entity.</returns>
+        /// <returns>The path as a string of the sub-queue entity.</returns>
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/EntityNameFormatter.cs:38]:
```suggestion
        /// <returns>The path as a string of the subqueue entity.</returns>
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/SubQueue.cs [hunk 23]
```diff
@@ -4,23 +4,23 @@
 namespace Azure.Messaging.ServiceBus
 {
     /// <summary>
-    /// Represents the possible system subqueues that can be received from.
+    /// Represents the possible system sub-queues that can be received from.
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/SubQueue.cs:7]:
```suggestion
    /// Represents the possible system subqueues that can be received from.
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/SubQueue.cs [hunk 24]
```diff
@@ -4,23 +4,23 @@
...
     /// </summary>
     public enum SubQueue
     {
         /// <summary>
-        /// No subqueue, the queue itself will be received from.
+        /// No sub-queue, the queue itself will be received from.
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/SubQueue.cs:12]:
```suggestion
        /// No subqueue, the queue itself will be received from.
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/SubQueue.cs [hunk 25]
```diff
@@ -4,23 +4,23 @@
...
         /// </summary>
         None = 0,
 
         /// <summary>
-        /// The dead-letter subqueue contains messages that have been deadlettered.
+        /// The dead-letter sub-queue contains messages that have been dead-lettered.
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/SubQueue.cs:17]:
```suggestion
        /// The dead-letter subqueue contains messages that have been dead-lettered.
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/SubQueue.cs [hunk 26]
```diff
@@ -4,23 +4,23 @@
...
         /// <see href="https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-dead-letter-queues#moving-messages-to-the-dlq"/>
         /// </summary>
         DeadLetter = 1,
 
         /// <summary>
-        /// The transfer dead-letter subqueue contains messages that have been deadlettered when
+        /// The transfer dead-letter sub-queue contains messages that have been dead-lettered when
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/SubQueue.cs:23]:
```suggestion
        /// The transfer dead-letter subqueue contains messages that have been dead-lettered when
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs [hunk 27]
```diff
@@ -603,15 +603,15 @@ private async Task ApplyPlugins(IReadOnlyList<ServiceBusReceivedMessage> message
         }
 
         /// <summary>
-        /// Moves a message to the deadletter sub-queue.
+        /// Moves a message to the dead-letter sub-queue.
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs:606]:
```suggestion
        /// Moves a message to the dead-letter subqueue.
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs [hunk 28]
```diff
@@ -603,15 +603,15 @@ private async Task ApplyPlugins(IReadOnlyList<ServiceBusReceivedMessage> message
...
         /// </summary>
         ///
-        /// <param name="message">The <see cref="ServiceBusReceivedMessage"/> to deadletter.</param>
+        /// <param name="message">The <see cref="ServiceBusReceivedMessage"/> to dead-letter.</param>
         /// <param name="propertiesToModify">The properties of the message to modify while moving to sub-queue.</param>
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs:610]:
```suggestion
        /// <param name="propertiesToModify">The properties of the message to modify while moving to subqueue.</param>
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs [hunk 29]
```diff
@@ -631,15 +631,15 @@ private async Task ApplyPlugins(IReadOnlyList<ServiceBusReceivedMessage> message
         }
 
         /// <summary>
-        /// Moves a message to the deadletter sub-queue.
+        /// Moves a message to the dead-letter sub-queue.
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs:634]:
```suggestion
        /// Moves a message to the dead-letter subqueue.
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs [hunk 30]
```diff
@@ -631,15 +631,15 @@ private async Task ApplyPlugins(IReadOnlyList<ServiceBusReceivedMessage> message
...
         /// </summary>
         ///
-        /// <param name="lockToken">The lock token of the <see cref="ServiceBusReceivedMessage"/> to deadletter.</param>
+        /// <param name="lockToken">The lock token of the <see cref="ServiceBusReceivedMessage"/> to dead-letter.</param>
         /// <param name="propertiesToModify">The properties of the message to modify while moving to sub-queue.</param>
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs:638]:
```suggestion
        /// <param name="propertiesToModify">The properties of the message to modify while moving to subqueue.</param>
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs [hunk 31]
```diff
@@ -656,18 +656,18 @@ private async Task ApplyPlugins(IReadOnlyList<ServiceBusReceivedMessage> message
                 cancellationToken: cancellationToken).ConfigureAwait(false);
 
         /// <summary>
-        /// Moves a message to the deadletter sub-queue.
+        /// Moves a message to the dead-letter sub-queue.
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs:659]:
```suggestion
        /// Moves a message to the dead-letter subqueue.
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs [hunk 32]
```diff
@@ -686,18 +686,18 @@ private async Task ApplyPlugins(IReadOnlyList<ServiceBusReceivedMessage> message
         }
 
         /// <summary>
-        /// Moves a message to the deadletter sub-queue.
+        /// Moves a message to the dead-letter sub-queue.
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs:689]:
```suggestion
        /// Moves a message to the dead-letter subqueue.
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs [hunk 33]
```diff
@@ -713,19 +713,19 @@ private async Task ApplyPlugins(IReadOnlyList<ServiceBusReceivedMessage> message
                 cancellationToken: cancellationToken).ConfigureAwait(false);
 
         /// <summary>
-        /// Moves a message to the deadletter sub-queue.
+        /// Moves a message to the dead-letter sub-queue.
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs:716]:
```suggestion
        /// Moves a message to the dead-letter subqueue.
```

File: sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiverOptions.cs [hunk 34]
```diff
@@ -37,7 +37,7 @@ public int PrefetchCount
         public ReceiveMode ReceiveMode { get; set; } = ReceiveMode.PeekLock;
 
         /// <summary>
-        /// Gets or sets the subqueue to connect the receiver to. By default, the receiver will not connect to a subqueue.
+        /// Gets or sets the sub-queue to connect the receiver to. By default, the receiver will not connect to a sub-queue.
```
Comment (MiYanni) [inline sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiverOptions.cs:40]:
```suggestion
        /// Gets or sets the subqueue to connect the receiver to. By default, the receiver will not connect to a subqueue.
```

File: sdk/core/Azure.Core.Experimental/src/Spatial/GeoBoundingBox.cs [hunk 35]
```diff
@@ -11,22 +11,22 @@ namespace Azure.Core.GeoJson
     public sealed class GeoBoundingBox : IEquatable<GeoBoundingBox>
     {
         /// <summary>
-        /// The westmost value of <see cref="GeoObject"/> coordinates.
+        /// The west-most value of <see cref="GeoObject"/> coordinates.
```
Comment (JoshLove-msft) [inline sdk/core/Azure.Core.Experimental/src/Spatial/GeoBoundingBox.cs:14]: Is the hyphen correct here - https://www.merriam-webster.com/dictionary/westmost

**Review by JoshLove-msft — APPROVED** [review]

File: sdk/core/Azure.Core.Experimental/src/Spatial/GeoBoundingBox.cs (hunk 35 above)
Comment (MiYanni) [reply sdk/core/Azure.Core.Experimental/src/Spatial/GeoBoundingBox.cs:14]:
```suggestion
        /// The westmost value of <see cref="GeoObject"/> coordinates.
```

File: sdk/core/Azure.Core.Experimental/src/Spatial/GeoBoundingBox.cs [hunk 36]
```diff
@@ -11,22 +11,22 @@ namespace Azure.Core.GeoJson
...
         /// </summary>
         public double West { get; }
 
         /// <summary>
-        /// The southmost value of <see cref="GeoObject"/> coordinates.
+        /// The south-most value of <see cref="GeoObject"/> coordinates.
```
Comment (MiYanni) [inline sdk/core/Azure.Core.Experimental/src/Spatial/GeoBoundingBox.cs:19]:
```suggestion
        /// The southmost value of <see cref="GeoObject"/> coordinates.
```

File: sdk/core/Azure.Core.Experimental/src/Spatial/GeoBoundingBox.cs [hunk 37]
```diff
@@ -11,22 +11,22 @@ namespace Azure.Core.GeoJson
...
         /// </summary>
         public double South { get; }
 
         /// <summary>
-        /// The eastmost value of <see cref="GeoObject"/> coordinates.
+        /// The east-most value of <see cref="GeoObject"/> coordinates.
```
Comment (MiYanni) [inline sdk/core/Azure.Core.Experimental/src/Spatial/GeoBoundingBox.cs:24]:
```suggestion
        /// The eastmost value of <see cref="GeoObject"/> coordinates.
```

File: sdk/core/Azure.Core.Experimental/src/Spatial/GeoBoundingBox.cs [hunk 38]
```diff
@@ -11,22 +11,22 @@ namespace Azure.Core.GeoJson
...
         /// </summary>
         public double East { get; }
 
         /// <summary>
-        /// The northmost value of <see cref="GeoObject"/> coordinates.
+        /// The north-most value of <see cref="GeoObject"/> coordinates.
```
Comment (MiYanni) [inline sdk/core/Azure.Core.Experimental/src/Spatial/GeoBoundingBox.cs:29]:
```suggestion
        /// The northmost value of <see cref="GeoObject"/> coordinates.
```
